tests/Formatter/XmlDataResponseFormatterTest.php

```diff
@@ -146,4 +126,76 @@ public function testFormatterWithContentType(): void
         );
         $this->assertSame(['text/xml; UTF-8'], $result->getHeader('Content-Type'));
     }
+
+    public function testFormatterWithUseTraversable(): void
```

Comment:
```suggestion
    public function testWithUseTraversable(): void
```

Seems all test names could be changed not to include `Formatter` after `test` since it is obvious that it's about formatter.

Reply:
rename